src/3rdparty/fmt/core.h

```diff
@@ -529,7 +529,6 @@ void check_format_string(S);
 
 struct error_handler {
   constexpr error_handler() = default;
-  constexpr error_handler(const error_handler&) = default;
```

Comment:
Probably better to fully update the library files.
See https://github.com/fmtlib/fmt/blob/master/include/fmt/core.h#L649-L650

Reply:
Oh, yeah. Forgot to write down that I had the exact same idea, and actually went ahead with it for a while but for some reasons I don't remember anymore it didn't really want to co-operate, so I stopped that attempt.
Then I'd rather go to `std::format`, but that requires C++20 and the compilers aren't quite up to speed yet.

Reply:
Ok, I updated the files, MSVC builds and regression pass, but gcc in wsl fails to build.